java/map-image-layer-tables/src/main/java/com/esri/arcgisruntime/sample/mapimagelayertables/MainActivity.java

```diff
@@ -0,0 +1,200 @@
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.mapimagelayertables;
+
+import android.graphics.Color;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+import android.widget.Toast;
+import com.esri.arcgisruntime.arcgisservices.RelationshipInfo;
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.data.ArcGISFeature;
+import com.esri.arcgisruntime.data.Feature;
+import com.esri.arcgisruntime.data.FeatureQueryResult;
+import com.esri.arcgisruntime.data.QueryParameters;
+import com.esri.arcgisruntime.data.RelatedFeatureQueryResult;
+import com.esri.arcgisruntime.data.RelatedQueryParameters;
+import com.esri.arcgisruntime.data.ServiceFeatureTable;
+import com.esri.arcgisruntime.geometry.Envelope;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.layers.ArcGISMapImageLayer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol;
+import com.esri.arcgisruntime.symbology.Symbol;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+public class MainActivity extends AppCompatActivity {
+
+  private static final String TAG = MainActivity.class.getSimpleName();
+
+  private MapView mMapView;
+  ListView list;
+  ArrayList<String> commentList;
+  ArrayList<Feature> featureList;
+  FeatureQueryResult commentQueryResult;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+
+    // inflate views from layout
+    mMapView = findViewById(R.id.mapView);
+    list = findViewById(R.id.comment_list);
+    // initialize list that will hold the comments for the list view
+    commentList = new ArrayList<>();
+    // initialize feature list that will hold the corresponding features for each comment
+    featureList = new ArrayList<>();
+    // create a map with a topographic basemap
+    ArcGISMap map = new ArcGISMap(Basemap.createStreets());
+
+    // create a new ArcGISMapImageLayer with a Service Request Map Server and load tables and layers
+    ArcGISMapImageLayer serviceRequestMapImageLayer = new ArcGISMapImageLayer(getString(R.string.map_service));
+    serviceRequestMapImageLayer.loadTablesAndLayersAsync();
+
+    // initialize graphics overlay
+    GraphicsOverlay graphicsOverlay = new GraphicsOverlay();
+
+    //TODO: check if initial viewpoint is working
```

Comment:
@TADraeseke can you check if the initial viewpoint is correct for you? for me it shows the whole map on the screen. I think it is only supposed to show North America and South America.

Reply:
also it does not have a README, the issue link does not have one so I am not too sure what to add.